src/Adapter/Supplier/QueryHandler/GetSupplierForViewingHandler.php

```diff
@@ -118,77 +117,116 @@ private function getSupplierProducts(Supplier $supplier, LanguageId $languageId)
         $products = [];
         $supplierProducts = $supplier->getProductsLite($languageId->getValue());
 
-        try {
-            foreach ($supplierProducts as $productData) {
-                $product = new Product($productData['id_product'], false, $languageId->getValue());
-                $product->loadStockData();
-
-                $productCombinations = $product->getAttributeCombinations($languageId->getValue());
-                $combinations = [];
-
-                foreach ($productCombinations as $combination) {
-                    $attributeId = $combination['id_product_attribute'];
-                    if (!isset($combinations[$attributeId])) {
-                        $productInfo = Supplier::getProductInformationsBySupplier(
-                            $supplier->id,
-                            $product->id,
-                            $combination['id_product_attribute']
-                        );
-                        $isoCode = Currency::getIsoCodeById((int) $productInfo['id_currency'])
-                            ?: $this->defaultCurrencyIsoCode;
-                        $formattedWholesalePrice = null !== $productInfo['product_supplier_price_te']
-                            ? $this->locale->formatPrice($productInfo['product_supplier_price_te'], $isoCode)
-                            : null;
-                        $combinations[$attributeId] = [
-                            'reference' => $combination['reference'],
-                            'supplier_reference' => $combination['supplier_reference'],
-                            'wholesale_price' => $formattedWholesalePrice,
-                            'ean13' => $combination['ean13'],
-                            'upc' => $combination['upc'],
-                            'quantity' => $combination['quantity'],
-                            'attributes' => '',
-                        ];
-                    }
-                    $attribute = sprintf(
-                        '%s - %s',
-                        $combination['group_name'],
-                        $combination['attribute_name']
-                    );
-
-                    if (!empty($combinations[$attributeId]['attributes'])) {
-                        $attribute = sprintf(', %s', $attribute);
-                    }
-
-                    $combinations[$attributeId]['attributes'] = $attribute;
-                }
+        foreach ($supplierProducts as $productData) {
+            $product = new Product($productData['id_product'], false, $languageId->getValue());
+            $product->loadStockData();
```

Comment:
I saw in another PR this method being deleted

Reply:
My bad, it was deleting this method for combinations
https://github.com/PrestaShop/PrestaShop/pull/26682/files#diff-a1cf5fd4d60c26d43efc6ffdad5f2606c95ca4bc50512ca4d92285ebcd687f0fL146

Reply:
Yes this part was not removed yet in Product